test/mocks/MockAppConfig.scala

```diff
@@ -50,9 +50,6 @@ class MockAppConfig(val runModeConfiguration: Configuration, val mode: Mode = Mo
   override val feedbackFormPartialUrl: String = "BasefeedbackUrl"
   override val contactFormServiceIdentifier: String = "VATVC"
   override val staticDateValue: String = "2018-05-01"
-  override val surveyUrl: String = "/some-survey-url"
```

Comment:
App currently does not compile. The relevant values need to be removed from AppConfig, FrontendAppConfig and ConfigKeys.

Reply:
@lewishayqa's comments still stand `surveyThankyou` and `surveyUrl` need to be removed from AppConfig, FrontendAppConfig or the tests don't compile. As is the tests are not running.